android/app/src/main/java/org/uwaterloo/subletr/navigation/NavigationDestination.kt

```diff
@@ -49,5 +49,11 @@ enum class NavigationDestination(
 			bottomBarNavigationTextId = R.string.chat,
 		),
 		showBottomBar = true,
-	)
+	),
+	LISTING_DETAILS(
+		rootNavPath = "listingdetails",
```

Comment:
If you want the "home" icon to be highlighted in the bottom bar, I think you need this to be "home/listingdetails"?